projects/1.18/assets/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+  "item.butchercraft.fried_chicken": "炸鸡块",
+  "item.butchercraft.fried_fish": "炸鱼",
+  "item.butchercraft.fries": "薯条",
+  "item.butchercraft.gelatin": "吉利丁",
+  "item.butchercraft.gravy_boat_item": "肉汁",
+  "item.butchercraft.grilled_cheese": "烤奶酪三明治",
+  "item.butchercraft.ground_beef": "生牛绞肉",
+  "item.butchercraft.ground_lamb": "生羊绞肉",
+  "item.butchercraft.ground_pork": "生猪绞肉",
+  "item.butchercraft.gut_knife": "穿肠刀",
+  "item.butchercraft.haggis": "碗装哈吉斯",
+  "item.butchercraft.hash": "碗装肉丁土豆",
+  "item.butchercraft.heart": "生心脏",
+  "item.butchercraft.horn": "角",
+  "item.butchercraft.jelly_black": "碗装黑莓果冻",
+  "item.butchercraft.jelly_blue": "碗装蓝莓果冻",
+  "item.butchercraft.jelly_brown": "碗装可乐果冻",
+  "item.butchercraft.jelly_cyan": "碗装潘趣果冻",
+  "item.butchercraft.jelly_green": "碗装苹果果冻",
+  "item.butchercraft.jelly_grey": "碗装神秘果冻",
+  "item.butchercraft.jelly_light_blue": "碗装蓝色覆盆子果冻",
+  "item.butchercraft.jelly_light_grey": "碗装火龙果果冻",
+  "item.butchercraft.jelly_lime": "碗装酸橙果冻",
+  "item.butchercraft.jelly_magenta": "碗装杂莓果冻",
+  "item.butchercraft.jelly_orange": "碗装橘子果冻",
+  "item.butchercraft.jelly_pink": "碗装草莓果冻",
+  "item.butchercraft.jelly_purple": "碗装葡萄果冻",
+  "item.butchercraft.jelly_red": "碗装樱桃果冻",
+  "item.butchercraft.jelly_white": "碗装椰子果冻",
+  "item.butchercraft.jelly_yellow": "碗装柠檬果冻",
+  "item.butchercraft.ketchup_jar_item": "番茄酱",
+  "item.butchercraft.kidney": "生肾脏",
+  "item.butchercraft.lamb_jerky": "羊肉干",
+  "item.butchercraft.lamb_rib": "生羊排骨",
+  "item.butchercraft.lamb_ribs": "生原切羊排骨",
+  "item.butchercraft.lamb_roast": "生的原切羊扒",
+  "item.butchercraft.lamb_scraps": "生羊肉丝",
+  "item.butchercraft.lamb_stew": "炖羊肉",
+  "item.butchercraft.lamb_stewmeat": "生羊碎肉",
+  "item.butchercraft.lard": "油脂",
+  "item.butchercraft.leather_cord": "皮绳",
+  "item.butchercraft.leather_scrap": "皮屑",
+  "item.butchercraft.liver": "生肝",
+  "item.butchercraft.liver_onions": "洋葱炒肝",
+  "item.butchercraft.lung": "生肺",
+  "item.butchercraft.mashed_potato_gravy": "碗装肉汁土豆泥",
+  "item.butchercraft.meat_pie_slice": "肉饼切片",
+  "item.butchercraft.oxtail": "生牛尾",
+  "item.butchercraft.oxtail_soup": "牛尾汤",
+  "item.butchercraft.pig_carcass": "待宰小猪",
+  "item.butchercraft.pork_jerky": "猪肉干",
+  "item.butchercraft.pork_rib": "生猪排骨",
+  "item.butchercraft.pork_ribs": "生原切猪排骨",
+  "item.butchercraft.pork_roast": "生的原切猪扒",
+  "item.butchercraft.pork_scraps": "生猪肉丝",
+  "item.butchercraft.pork_stew": "炖猪肉",
+  "item.butchercraft.pork_stewmeat": "生猪碎肉",
+  "item.butchercraft.pork_tenderloin": "猪里脊",
+  "item.butchercraft.potroast": "碗装罐焖牛肉",
+  "item.butchercraft.pulled_pork_sandwich": "手撕猪肉三明治",
+  "item.butchercraft.rack_lamb": "盘装羊排",
+  "item.butchercraft.salisbury_steak": "盘装索尔兹伯里牛排",
+  "item.butchercraft.sausage": "生香肠",
+  "item.butchercraft.sausage_roll": "香肠卷",
+  "item.butchercraft.sheep_carcass": "待宰羔羊",
+  "item.butchercraft.sinew": "肌腱",
+  "item.butchercraft.skinning_knife": "剥皮刀",
+  "item.butchercraft.sos": "SOS",
```

```suggestion
  "item.butchercraft.sos": "奶油牛肉丝面包",
```
https://www.aspicyperspective.com/creamed-chipped-beef-on-toast-shit-on-a-shingle/
即`Shit on a Shingle`（美国海军俚语）的缩写，正式称呼是`cream chipped beef on toast`
这玩意还是翻译出来比较好？